common/oauthTokenManager.go

```diff
@@ -163,23 +163,27 @@ func (uotm *UserOAuthTokenManager) validateAndPersistLogin(oAuthTokenInfo *OAuth
 }
 
 func (uotm *UserOAuthTokenManager) AzCliLogin(tenantID string) error {
+
+	// CLI creds will not be persisted. AzCLI would have already persistd that
 	oAuthTokenInfo := &OAuthTokenInfo{
```

Comment:
In case of CLI login we want to persist the token then why flag is false here.

Reply:
the comment above it is explaining the behavior observed here in this case , in our original flow also persist flag for AzCliLogin was kept as false therefore to maintain consistency we have passed Persist as false here.